src/utils/commandRunner.ts

```diff
@@ -31,7 +31,7 @@ export class CommandRunner {
   }> {
     let executablePath: string;
     let command: string;
-    let runEnv: NodeJS.ProcessEnv | undefined;
```

Comment:
Doesn't `undefined` have different semantics?
It's passed to the subprocess invocation unconditionally. Dunno about nodejs but in Python, if you don't pass an explicit env value, then it is inherited from the parent process, while setting a dict would replace the env vars exposed to the subprocess. So setting an empty dict means removing all of the available environment variables which will likely have disastrous consequences.

Reply:
That's correct. See: https://nodejs.org/api/child_process.html#child_processexeccommand-options-callback